src/app/shared/plnkr/template/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
```

Comment:
I still advocate moving the template files to assets, they're not really part of the application source